### PR TITLE
Rough in "charter" content in README.

### PR DESCRIPTION
**Pull request by dblodgett-usgs**
Per the discussion in #11 -- I think this is where we've landed for how we are going to jump into this.

In discussion with Jonathan Gregory, I've been advised that our goals around origin/offset style coordinates are not actually out of line with the CF data model documented in Appendix I and that we should probably just go ahead using that CF data model and implementing the compact origin/offset style coordinate variables needed for direct geotiff compatibility as an implementation of the CF data model seperate from the NetCDF-CF convention. I've noted that in deliverables. 

I'm not attached to any of this but wanted to move us along since it seems we are in agreement.

### CODE REVIEW

**Comment by dblodgett-usgs** [comment]
Per the conversation at the GeoZarr spec call today, what I have in this PR around the deliverable is not necessarily correct. For now, we are just going to evolve https://github.com/zarr-developers/geozarr-spec/blob/main/geozarr-spec.md to accommodate the full suite of requirements of geozarr. How this is published as a normative standard is an open question. I will try to summarize where I think the group stands regarding that in an update to this PR.

**Comment by dblodgett-usgs** [comment]
@briannapagan @christophenoel @rabernat -- I think this now reflects where we stand?

https://github.com/zarr-developers/geozarr-spec/pull/13/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

**Comment by christophenoel** [comment]
Seems very good to me

**Comment by briannapagan** [comment]
Yes, I will also commit some minor changes. Thanks David!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+* Allowing scientists and researchers to work with diverse data types and projections in their preferred software and programming languages.
+
+## Deliverable
+
+The geozarr-spec is a community effort to establish conventions for geospatial use cases in the cloud-native ZARR binary carrier. It is built from existing data standards, conventions, and implementation patterns. The outcome of this work will be an implementable set of conventions. Next steps for formalization of a normative version of the conventions will be refined as the work progresses. Some key considerations follow.
+
+* A ZARR Convention according to the draft described in [Zarr Enhancement Proposal 0004](https://github.com/zarr-developers/zeps/pull/28/files?short_path=53e442a#diff-53e442aa938ca18ba1a94f845f264c5df0e4650f8f6abda856c3cd819f70abca). 
+* An OGC standard that describes the GeoZarr-spec convention. ([See two track standards process here for requirements of a full vs community standard](https://docs.opengeospatial.org/pol/05-020r27/05-020r27.html#the-two-track-standards-process-characteristics))
+* Contribution of any extension that is best housed in the CF-Convention.
```
[the remark this answers is not in the document]
Comment (dblodgett-usgs) [reply README.md:20]: This sentence isn't very good... sorry. I'll just type out the logic and see if a better summary sentence emerged. 

We are building a convention that layers on top of (extends) the CF baseline using the ZARR format to express the outcome. If we come up with something that extends but is actually a natural fit for CF-NetCDF, we would propose it back to the CF community such that there is a better interoperability arrangement between GeoZarr and CF-NetCDF. 

So...

* Proposed modifications to CF-NetCDF that foster greater interoperabilty between the GeoZarr-spec and CF-NetCDF convetnions.

Is that making sense?

**Pull request closed without merging**